data/mods/gen4/abilities.js

```diff
@@ -133,6 +133,38 @@ let BattleAbilities = {
 		inherit: true,
 		rating: 2.5,
 	},
+	"intimidate": {
+		inherit: true,
+		onStart(pokemon) {
+			let activated = false;
+			for (const target of pokemon.side.foe.active) {
+				if (target && this.isAdjacent(target, pokemon) &&
+					!(target.volatiles['substitute'] ||
+						target.volatiles['substitutebroken'] && target.volatiles['substitutebroken'].move === 'uturn')) {
+					activated = true;
+					break;
+				}
+			}
+
+			if (!activated) {
+				this.hint('In Gen 4, Intimidate does not activate if every target has a Substitute (or the Substitute was just broken by U-turn).', false, pokemon.side.id);
```

Comment:
Heh, I guess we'll have to override Intimidate in Gen 3 or this won't make much sense.

Reply:
Does Gen 3 intimidate have the same behavior re: not activating if the target has a sub, or was that just a Gen 4 regression (obviously, 'Gen 4' and 'U-turn' mean that this hint is definitely out of place so thanks for catching that, I'm just trying to figure out what the correct behavior for Gen 3 should be)

Reply:
Same behaviour, yeah. Might as well cut out the `substitutebroken` stuff from there as well, but I don't think it would be a problem either way.

Reply:
Done.